4-Express/server/handlers/storage/getAllItems.js

```diff
@@ -0,0 +1,7 @@
+const path = require('path')
+const db = require(path.resolve('server/db/','db')) 
```

Comment:
why not `require(server/db/db)`? you can also name it `index.js` and it will require it by default